code/API_definitions/location-retrieval.yaml

```diff
@@ -466,23 +471,30 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 403
+                  code:
+                    enum:
+                      - PERMISSION_DENIED
+                      - INVALID_TOKEN_CONTEXT

```

Comment:
We may need to remove INVALID_TOKEN_CONTEXT for our APIs. There is an ongoing discussion in Commonalities

Reply:
I tend to prefer to remove it. What is your view @jlurien as we need probably to move forward with this PR? I remove it, we merge this one and we listen the Commonalities version in case we need to reintroduce it?